CMakeLists.txt

```diff
@@ -27,21 +26,19 @@ set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/CMake"
 
 find_package(Sanitizers REQUIRED)
 
+include(GTest)
 include(boost)
 include(Eigen)
 include(pybind11)
-include(GoogleTest)
 include(GoogleBenchmark)
 include(GenerateExportHeader)
+find_package(Boost REQUIRED)
```

Comment:
Any reasons for dropping the `1.67` requirement? We do actually need that.

Reply:
Yes, that can be added back.